pkg/util/util.go

```diff
@@ -721,3 +733,30 @@ func PrintThisNoteWhenFinishedApplyingAndStartWaitLoop() {
 	log.Printf("  - From now on, it only loops and reads the status, to monitor the operator work.")
 	log.Printf("  - You may Ctrl-C at any time to stop the loop and watch it manually.")
 }
+
+func DiscoverOAuthEndpoints() (*OAuth2Endpoints, error) {
```

Comment:
I see that oauth-proxy has the same logic here:
https://github.com/openshift/oauth-proxy/blob/fbd062bbcb23538863a735f9823ac3fd6f47429e/providers/openshift/provider.go#L543-L567

Do you think we can/should call that code instead?

Reply:
We are not using OAuth proxy or its codebase